misk-aws/src/main/kotlin/misk/dynamodb/RealDynamoDbModule.kt

```diff
@@ -0,0 +1,46 @@
+package misk.dynamodb
+
+import com.amazonaws.auth.AWSCredentialsProvider
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig
+import com.google.inject.Provides
+import misk.cloud.aws.AwsRegion
+import misk.inject.KAbstractModule
+import javax.inject.Singleton
+
+internal class RealDynamoDbModule : KAbstractModule() {
+  override fun configure() {
+    requireBinding<AWSCredentialsProvider>()
+    requireBinding<AwsRegion>()
+  }
+
+  @Provides @Singleton
+  fun providesAmazonDynamoDB(
+    awsRegion: AwsRegion,
+    awsCredentialsProvider: AWSCredentialsProvider
+  ): AmazonDynamoDB {
+    return AmazonDynamoDBClientBuilder
+        .standard()
+        .withRegion(awsRegion.name)
+        .withCredentials(awsCredentialsProvider)
+        .build()
+  }
+
+  @Provides @Singleton
+  fun providesDynamoDBmapper(dynamoDb: AmazonDynamoDB): DynamoDBMapper {
+    return dynamoDbMapper(dynamoDb)
+  }
+
+  companion object {
+    internal fun dynamoDbMapper(dynamoDb: AmazonDynamoDB): DynamoDBMapper {
+      return DynamoDBMapper(
+          dynamoDb,
+          DynamoDBMapperConfig.Builder()
+              .withSaveBehavior(DynamoDBMapperConfig.SaveBehavior.CLOBBER)
```

Comment:
Is this an option that should be exposed via a config?

Reply:
After some slack discussion, yes

Reply:
I was thinking about whether this should be available from a config file, or just as a parameter for the Module. But then I wondered if the module should even be providing this at all..

If we ever wanted the possibility of having 2 tables with different save behaviors, this wouldn't work. Is it better to leave this configuration to the app @mightyguava ?

Reply:
Yeah, it sounds like this doesn't belong in the module